funcx_web_service/models/tasks.py

```diff
@@ -119,149 +70,102 @@ class Task:
 
     def __init__(
         self,
-        rc: StrictRedis,
+        redis_client: Redis,
         task_id: str,
-        user_id: int = -1,
-        function_id: str = "",
-        container: str = "",
-        serializer: str = "",
-        payload: str = "",
-        task_group_id: str = ""
+        *,
+        user_id: t.Optional[int] = None,
+        function_id: t.Optional[str] = None,
+        container: t.Optional[str] = None,
+        payload: t.Any = None,
+        task_group_id: t.Optional[str] = None,
     ):
-        """ If the kwargs are passed, then they will be overwritten.  Otherwise, they will gotten from existing
-        task entry.
-        Parameters
-        ----------
-        rc : StrictRedis
-            Redis client so that properties can get get/set
-        task_id : str
-            UUID of task
-        user_id : int
-            ID of user that this task belongs to
-        function_id : str
-            UUID of function for task
-        container : str
-            UUID of container in which to run
-        serializer : str
-        payload : str
-            serialized function + input data
-        task_group_id : str
-            UUID of task group that this task belongs to
         """
-        self.rc = rc
+        If optional values are passed, then they will be written.
+        Otherwise, they will fetched from any existing task entry.
+
+        :param redis_client: Redis client for properties to get/set
+        :param task_id: UUID of the task, as str
+        :param user_id: ID of user to whom this task belongs
+        :param function_id: UUID of the function for this task, as str
+        :param container: UUID of container in which to run, as str
+        :param payload: serialized function + input data
+        :param task_group_id: UUID of task group that this task belongs to
```

Comment:
Style changes don't matter very much, but it's nice to be consistent. It might be better to make sure folks are okay with the change and then do updates wholesale.